test/vc-data-model-1.0/50-jwt.js

```diff
@@ -60,7 +60,7 @@ describe('JWT (optional)', () => {
         const jwtResult = new jwt.JwsToken(jwtBase64);
         expect(jwtResult.isContentWellFormedToken()).to.be.true;
 
-        const typ = jwtResult.getHeader('typ');
+        const typ = jwtResult.getHeader()['typ'];
```

Comment:
a better javascript way to do this would be to use destructure syntax like so:
`const { typ } = jwtResult.getHeader()`